Cabal/src/Distribution/Simple/Setup/Build.hs

```diff
@@ -65,6 +66,7 @@ defaultBuildFlags =
     , buildDistPref = mempty
     , buildVerbosity = Flag normal
     , buildNumJobs = mempty
+    , buildUseSemaphore = NoFlag
```

Comment:
`NoFlag` or `Serial`?

Reply:
Not sure if there's a major practical difference. Let's keep it like that for now.